### PR TITLE
Small usability improvement to pitchshifter

### PR DESCRIPTION
**Pull request by dahlgrenmartin**
Hi! 

I hope this is okey as a fix for my own issue:
https://github.com/godotengine/godot/issues/28461#issue-437923191

Cheers,
Martin.

### CODE REVIEW

**Comment by YeldhamDev** [comment]
Please squash your commits: https://docs.godotengine.org/en/3.1/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

File: servers/audio/effects/audio_effect_pitch_shift.cpp [hunk 1]
```diff
@@ -28,6 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+#include <cmath>
```
Comment (akien-mga) [inline servers/audio/effects/audio_effect_pitch_shift.cpp]: I don't see any use of `<cmath>`?

File: servers/audio/effects/audio_effect_pitch_shift.h [hunk 2]
```diff
@@ -79,17 +79,32 @@ class AudioEffectPitchShiftInstance : public AudioEffectInstance {
...
+	};
 
+public:
 	friend class AudioEffectPitchShiftInstance;
 
 	float pitch_scale;
-	int window_size;
+	int oversampling;
+	FFT_Size fft_size;
+	int window_size; // Not used... could be removed, right?
```
Comment (akien-mga) [inline servers/audio/effects/audio_effect_pitch_shift.h]: Yes, please remove it.

File: servers/audio/effects/audio_effect_pitch_shift.cpp (hunk 1 above)
Comment (mdahlgrengadd) [reply servers/audio/effects/audio_effect_pitch_shift.cpp]: No sorry, old used in my old code

File: servers/audio/effects/audio_effect_pitch_shift.h (hunk 2 above)
Comment (mdahlgrengadd) [reply servers/audio/effects/audio_effect_pitch_shift.h]: ok

**Comment by YeldhamDev** [comment]
Remember to squash it again.

**Comment by mdahlgrengadd** [comment]
@YeldhamDev Sorry, i thought i did... will try again.

**Comment by bojidar-bg** [comment]
Ouch. Seems like you rebased in the wrong direction. Maybe try `git reflog` to get the old commit, then `git checkout` or `git reset` to get to it, and finally `git rebase master`.

**Comment by reduz** [comment]
As soon as @akien-mga approves form, this is fine.

**Comment by mdahlgrengadd** [comment]
@Ibojidar-bg Reflog does nothing, maybe I have to be on the computer were I messed up? Dont have access to it until next week, though.

 I'm thinking of just wiping this repository and start fresh. I'm spending more time on git than on the actual code.

**Comment by mdahlgrengadd** [comment]
@akien-mga I've reset my repo, do I need to do another pull request now?

**Comment by akien-mga** [comment]
@mdahlgrengadd Yes, this PR now tracks `unknown repository` so it can't be updated further.

**Pull request closed without merging**